Client/Extensions/UIAlertControllerExtensions.swift

```diff
@@ -118,4 +118,49 @@ extension UIAlertController {
         alert.addAction(noOption)
         return alert
     }
+
+    /**
+     Creates an alert view to warn the user that their logins will either be completely deleted in the 
+     case of local-only logins or deleted across synced devices in synced account logins.
+
+     - parameter deleteCallback: Block to run when delete is tapped.
+     - parameter hasAccount: Boolean indicating the user has a FxA account connected.
```

Comment:
So on desktop, I can configure what I want to sync so it's not enough to know if they have an account, I also need to know if they've enabled password syncing. 

Looking through the code it seems this configuration is part of the 'sync' prefs branch:

https://github.com/mozilla/firefox-ios/blob/master/Providers/Profile.swift#L685

Looks like if metaGlobal scratchpad doesn't include the 'passwords' collection, that means that password sync is not enabled (inferred from: https://github.com/mozilla/firefox-ios/blob/master/Sync/Synchronizers/Synchronizer.swift#L177) and this scratchpad lives inside the SyncStateMachine.

What would be the best way to expose this?